python/tests/server/fixtures/openapi_output_type.py

```diff
@@ -4,7 +4,7 @@
 
 # An output object called `Output` needs to be special cased because pydantic tries to dedupe it with the internal `Output`
 class Output(BaseModel):
-    foo_number: int = "42" # this can't be right... it's a string?
```

Comment:
FYI: In very JavaScript ["wat"](https://www.destroyallsoftware.com/talks/wat) fashion, Pydantic automatically casts "42" into the int 42